src/w3c/level.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * This module updates the title and shortName
+ * of a spec when there is a valid level configuration.
+ *
+ * Levels should be integers >= 0.
+ */
+
+import { getIntlData, showInlineWarning } from "../core/utils.js";
+export const name = "w3c/level";
+
+const localizationStrings = {
+  en: {
+    level: "Level",
+  },
+};
+
+const l10n = getIntlData(localizationStrings);
+
+export function run(conf) {
+  if (!conf.hasOwnProperty("level")) return;
+
+  const h1Elem = document.querySelector("h1#title");
+
+  const level = parseInt(conf.level);
+  if (!Number.isInteger(level) || level < 0) {
+    showInlineWarning(
+      h1Elem,
+      `The \`level\` configuration option must be a number greater or equal to 0. It is currently set to \`${level}\``,
+      "Invalid level config."
+    );
+    return;
+  }
+
+  h1Elem.append(` ${l10n.level} ${level}`);
+  document.title = `${document.title} ${l10n.level} ${level}`;
+  conf.shortName = `${conf.shortName}-${level}`;
```

Comment:
Let's finally use the canonical form, in case we need it again. 
 
```suggestion
  conf.level = level;
```

Reply:
by this you mean that you just want to add an additional line that says `conf.level = level;` right? 

The suggestion also says that you want to remove `conf.shortName = `${conf.shortName}-${level}`;`

Reply:
oh whoops! yes... argh. let me fix that!